packages/wallets/src/components/WalletsAccordion/WalletsAccordion.tsx

```diff
@@ -1,46 +1,52 @@
-import React, { ReactElement, useEffect } from 'react';
-import { useAuthorize, useWalletAccountsList } from '@deriv/api';
+import React, { useEffect } from 'react';
 import IcDropdown from '../../public/images/ic-dropdown.svg';
 import './WalletsAccordion.scss';
 
 type TProps = {
-    account: NonNullable<ReturnType<typeof useWalletAccountsList>['data']>[number];
-    content: ReactElement;
-    header: ReactElement;
+    isDemo?: boolean;
+    isOpen?: boolean;
+    onToggle?: () => void;
+    renderHeader: () => React.ReactNode;
 };
 
-const WalletsAccordion: React.FC<TProps> = ({ account: { is_active, is_virtual, loginid }, content, header }) => {
-    const { switchAccount } = useAuthorize();
+const WalletsAccordion: React.FC<React.PropsWithChildren<TProps>> = ({
+    children,
+    isDemo = false,
+    isOpen = false,
+    onToggle,
+    renderHeader,
+}) => {
     const accordionRef = React.useRef<HTMLDivElement>(null);
 
     useEffect(() => {
-        if (is_active && accordionRef?.current) {
+        if (isOpen && accordionRef?.current) {
             accordionRef.current.style.scrollMarginTop = '24px';
             accordionRef.current.scrollIntoView({ behavior: 'smooth' });
         }
-    }, [is_active]);
+    }, [isOpen]);
 
     return (
         <div
             className={`wallets-accordion wallets-accordion ${
-                is_virtual ? 'wallets-accordion wallets-accordion--virtual' : ''
+                isDemo ? 'wallets-accordion wallets-accordion--virtual' : ''
             }`}
+            ref={accordionRef}
```

Comment:
Was this even working before without the ref being applied? Did it fix a bug or was it assigned elsewhere? If so, nice.

Reply:
@markwylde-deriv Yeah, it was not working since the ref was not used 😅 Now is fixed.